src/TreeView/index.tsx

```diff
@@ -367,6 +369,7 @@ const treeReducer = (
 
 interface IUseTreeProps {
   data: INode[];
+  controlledIds?: number[];
```

Comment:
@kpustakhod You had named this prop `checkedIds` in our repo...

Reply:
@mellis481 I named it `checkedIds` first, then it was renamed to `controlledIds`.

Reply:
@kpustakhod I would expect our repo and this repo to be identical. Why was it not updated in our repo?

Reply:
Prop names were updated in separate PR - https://github.com/vertexinc/react-accessible-treeview/pull/16/files